README_WINDOWS.rst

```diff
@@ -0,0 +1,21 @@
+Borg Nativ on Windows
+=====================
+
+Build Requirements
+------------------
+
+- VC 14.0 Compiler
+- OpenSSL Library (https://slproweb.com/products/Win32OpenSSL.html)
+- Additional Python Packages:
+  - psutils
+- Patience and a lot of coffee / beer
+
+What's working
+--------------
+
+ - Borg does not crash if called with ``borg``
+ - ``borg init --encryption none ./demoRepo`` runs without an error/warning.
+   Note that only relative paths work at the moment.
+
+
```

Comment:
although it might seem more work at first, i guess we could avoid doing the work twice by:

- having windows specific instructions as part of the borg docs, see docs/...
- having bugs / todo in the issue tracker (maybe after the first merge)

Reply:
I think it's easier (until the first merge) to keep track of whats working/failing in this file. After the first merge we should integrate everything into the existing docs and use the issue tracker.